exercises/matrix_determinant/python/tests.py

```diff
@@ -0,0 +1,44 @@
+import unittest
```

Comment:
You should add unit tests for the functions getting user input acting the way you want, you will need to "mock" the `input` (and maybe `print`) functions.

https://stackoverflow.com/questions/47690020/python-3-unit-tests-with-user-input